include/ddc/parallel_for_each.hpp

```diff
@@ -150,7 +150,6 @@ void parallel_for_each(
 }
 
 /** iterates over a nD domain using the `Kokkos` default execution space
- * @param[in] domain the domain over which to iterate
```

Comment:
It is expected that this doesn't raise an error as I only check files in include/ddc/kernels

Reply:
in the CI yes but `doxygen.log` should have a line about it. I have documented the `ddc` namespace so that it should work when `EXTRACT_ALL` is `NO`.